packages/test/test-gc-sweep-tests/src/sendOps.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRandom } from "@fluid-internal/stochastic-test-utils";
+import { assert, delay } from "@fluidframework/common-utils";
+import { DataObjectWithCounter } from "./dataObjectWithCounter";
+
+export function start(dataObjectWithCounter: DataObjectWithCounter, count: number, random: IRandom) {
```

Comment:
Not sure I understand why these functions are global and not part of the data object. Can you please explain how the overall flow will work?
Here's how I am imagining the test to work which will confine all the logic of a data store within its code:
1. A data object creates and initializes another data object. No activity starts as part of this.
2. A data object references another data object and calls startActivity on it to start activity. The activity is the referenced data store sends op regularly and does something interesting after every few ops.
3. A data object calls stopActivity on another data object and unreferences it. The unreferenced data object stops sending ops and ceases to do any activity.

Reply:
This was split globally to not conflict with other files on merge.